dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -1,10 +1,13 @@
-import csv
-from sqlalchemy.exc import IntegrityError
-from dataactvalidator.filestreaming.fieldCleaner import FieldCleaner
-from dataactvalidator.validation_handlers.validator import Validator
+from datetime import datetime
 
 class LoaderUtils:
 
+    # define some data-munging functions that can be applied to
+    # pandas dataframes as necessary
+    padFunction = lambda field, padTo: str(field).strip().zfill(padTo)
+    currentTimeFunction = lambda x: datetime.utcnow()
+    cleanColNamesFunction = lambda field: str(field).lower().strip().replace(" ","_").replace(",","_")
```

Comment:
Any reason these are static variables as opposed to straight functions? They don't seem too tightly coupled with `LoaderUtils` and a `def` is generally easier to read than a `lambda`